README.md

```diff
@@ -19,10 +29,41 @@ private static IAsyncPolicy<HttpResponseMessage> GetCircuitBreakerPolicy()
 }
 ```
 
-3. In your standard `Startup.ConfigureServices(...)` method, start by configuring a named client as below:
+In public method `ConfigureServices` in `Startup.cs` file where **HTTPClient** is implementing add:
 ```csharp
 services.AddHttpClient("ProfileHttpClient")
     .AddPolicyHandler(GetRetryPolicy())
     .AddPolicyHandler(GetCircuitBreakerPolicy());
-where ProfileHttpClient - your HTTP client implementation
-```
\ No newline at end of file
+```
+## Usage
+Declare **HTTPClient**
+
+```csharp
+_httpClient = new RestHttpClient(new TestHttpClientFactory());
+```
+Send and get response
+```csharp
+_httpClient.SendAndGetResponseAsync(request)
```

Comment:
Need to add request example to demonstrate how to create it.

Reply:
I think it's not resolved because you didn't add implementation of UsersListRequest.